### DISCUSSION TITLE
WPF - CefSharp application not launching in Windows Server 2019 Standard Operating System

### DISCUSSION THREAD

**Discussion by gopinathsmart** [cefsharp/CefSharp#3328] [Q&A]
### Bug Report
I can't able to run my WPF CefSharp application in Windows Server 2019 OS.

- **What version of the product are you using?**
    86.0.241

- **What architecture x86 or x64?**
    x86
    
- **On what operating system?**
    Windows Server 2019 Standard

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
   WPF

- **System Requirements:**
1.  .NET Framework: 4.5.2
2. Processer: Intel(R) Core(TM) i7-8700 CPU @3.20GHz 3.19 GHz
3. Installed memory (RAM): 2.00 GB
4. System type : 64-bit Operating System, x64-based processor
5. Pen and Touch: No Pen or Touch Input is available for this Display.
    
- **What steps will reproduce the problem?**

1. Create a simple CefSharp Application with above requirements.
2. Load a simple HTML by Chromium Browser.
3. Build and Launch the .exe file in bin location.
4. It doesn't launched.

- **What is the expected output? What do you see instead?**
**_Expected Output:_**
Application needs to launch and show helloworld text.
![image](https://user-images.githubusercontent.com/22271799/101365994-74db2400-38ca-11eb-8d60-0828c5d5fc27.png)
**_Actual Output:_**
It doesn't launched.
![image](https://user-images.githubusercontent.com/22271799/101365551-f5e5eb80-38c9-11eb-8f76-9b3d92f6d2f5.png)

- **Please provide any additional information below.**

I created a simple repo here: https://github.com/gopinathsmart/CefSharp

- **Does this problem also occur in the `CEF` Sample Application**
Yes

Thanks,
Gopinath B

**Comment by amaitland** [comment]
Did you check the log file?

> Does this problem also occur in the CEF Sample Application
> Yes

If cefclient doesn't run then this isn't a bug in CefSharp,  most likely you are missing a required dependency. Does Chrome run?

**Comment by amaitland** [comment]
Moving conversation to new discussions section, might be a bit of noise
